pytorch_lightning/trainer/distrib_data_parallel.py

```diff
@@ -559,9 +560,13 @@ def ddp_train(self, process_idx, q, model, is_master=False, proc_offset=0):
         torch.cuda.empty_cache()
 
         if self.global_rank == 0 and q is not None:
+            rank_zero_warn('cleaning up ddp environment...')
             q.put(self.checkpoint_callback.best_model_path)
```

Comment:
could we add a None check here for checkpoint_callback? because the user can set it to None, if they want. 
See #2547

Reply:
fixed!